pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -186,6 +193,10 @@ func (br *OVSBridge) create() Error {
 	}
 
 	br.uuid = res[0].UUID[1]
+	br.isHardwareOffload, err = br.getHardwareOffload()
```

Comment:
As it needs to fetch hardwareOffload config regardless of `create` or `updateBridgeConfiguration` and the config is not related to bridge existence, should it be moved to `Create` to reduce code redundance?